SETUP.md

```diff
@@ -0,0 +1,154 @@
+# Sustineo Setup Guide
+
+Sustineo is an application that combines AI capabilities with real-time interactions for voice and image generation.
+
+## Prerequisites
+
+- [Python 3.11](https://www.python.org/downloads/) or later for the API
+- [Node.js 20](https://nodejs.org/) or later for the web component
+- Azure subscription with the following services:
+  - Azure OpenAI service (for voice capabilities)
+  - Azure OpenAI service (for image generation)
+  - Azure Cosmos DB account
+  - Azure Blob Storage account
+
+## Environment Setup
+
+### API Environment Variables
+
+Create a `.env` file in the root directory with the following variables:
+
+```
+# Azure OpenAI for Voice
+AZURE_VOICE_ENDPOINT=https://your-voice-openai-resource.openai.azure.com
+AZURE_VOICE_KEY=your-voice-openai-key
+
+# Azure OpenAI for Image Generation
+AZURE_IMAGE_ENDPOINT=https://your-image-openai-resource.openai.azure.com
+AZURE_IMAGE_API_KEY=your-image-openai-key
+
+# Azure Storage
+SUSTINEO_STORAGE=https://yourstorageaccount.blob.core.windows.net
+
+# Azure Cosmos DB
+COSMOSDB_CONNECTION=your-cosmosdb-connection-string
+
+# Optional: Azure Foundry (if using)
+FOUNDRY_CONNECTION=your-foundry-connection-string
+
+# Optional: Local tracing
+LOCAL_TRACING_ENABLED=false
+```
+
+### Web Environment Setup
+
+The web component uses the API endpoints defined in `web/store/endpoint.ts`. By default, these are set to:
+- WebSocket Endpoint: `ws://localhost:8000`
+- API Endpoint: `http://localhost:8000`
+- Web Endpoint: `http://localhost:5173`
```

Comment:
You can mention here that there is a GitHub Action in this project that rewrites these to match the Azure Container App by using the Azure CLI to query for the correct FQDNs